orc8r/cloud/go/services/certifier/certifier/main.go

```diff
@@ -83,6 +87,20 @@ func main() {
 		caMap[protos.CertType_VPN] = &servicers.CAInfo{Cert: vpnCert, PrivKey: vpnPrivKey}
 	}
 
+	var serviceConfig certifier.Config
+	_, _, err = config.GetStructuredServiceConfig(orc8r.ModuleName, certifier.ServiceName, &serviceConfig)
+	if err != nil {
+		glog.Errorf("err %v failed parsing the config file: skipping CollectorServicer creation ", err)
```

Comment:
Currently let's make this glog fatal and move the else out of the conditional. When it is desirable to make this optional, we can make this change accordingly

Reply:
The thought here was that this is only used for metrics, so if this fails for some reason, the entire `certifier` service shouldn't fail to run

Reply:
Given that we are collecting these metrics unconditionally and we always expect this file to be around, I think adding this assertion should help us catch any issues early on. If we return with an error, we might fail silently and never realize that the alert metrics never got registered. I'm favor of failing hard than failing silently.

Reply:
We might need to discuss this as a team, since we're removing assertFatal in MME, which is going in the opposite direction

Reply:
IMO we need the equivalent of a "File an issue in Github if this ever happens"

Reply:
I think MME is fundamentally different problem than orc8r. We can certainly discuss this. Please initiate the conversation.